docs/mkdocs.yml

```diff
@@ -148,7 +130,25 @@ nav:
         - pcf init: 'cmd/pa/pcf/pcf-init.md'
       - solution:
         - solution init: 'cmd/pa/solution/solution-init.md'
-        - solution reference add: 'cmd/pa/solution/solution-reference-add.md'
+        - solution reference add: 'cmd/pa/solution/solution-reference-add.md'        
+     - Power Automate (flow):
```

Comment:
Indented one space top much. Breaks building our docs 😁
Fixed it when merging changes 👍